### ISSUE TITLE
Mask not working on Safari/iOS

### ISSUE THREAD

**Issue by adrienlamotte** [ngx-lottie/ngx-lottie#24]
Hello ! 

My animation works fine outside of Angular, but used with ngx-lottie the mask are not preserved.
I did some research, and found out that issue : https://github.com/airbnb/lottie-web/issues/1198

So maybe you have the same issue here ? The solution would be : https://github.com/chenqingspring/ng-lottie/pull/35

Thanks for your work ;-)

**Comment by arturovt** [comment]
@adrienlamotte hey. Didn't know about that issue. Will investigate and publish a fix :stuck_out_tongue_closed_eyes:

**Comment by arturovt** [comment]
@adrienlamotte I've published the `4.1.2` version. Could you try to upgrade?
```console
yarn upgrade ngx-lottie@4.1.2
```

**Comment by adrienlamotte** [comment]
Hello @arturovt , I upgraded to 4.1.2 but now it triggers an error that block the page load : 

```
ERROR Error: Uncaught (in promise): NullInjectorError: StaticInjectorError(de)[le -> Qn]: 
  StaticInjectorError(Platform: core)[le -> Qn]: 
    NullInjectorError: No provider for Qn!
NullInjectorError: StaticInjectorError(de)[le -> Qn]: 
  StaticInjectorError(Platform: core)[le -> Qn]: 
    NullInjectorError: No provider for Qn!
```

Here is the animation I use : 
[performances.zip](https://github.com/ngx-lottie/ngx-lottie/files/3651024/performances.zip)

**Comment by arturovt** [comment]
@adrienlamotte what was the last version you were using before you upgraded?

**Comment by adrienlamotte** [comment]
@arturovt yes I noticed that I was in 3.1.0, did not saw the changes on the module import :-)
Now it works, and the Safari bug is gone !

Thanks for your quick release ;-)

**Comment by arturovt** [comment]
@adrienlamotte thanks for confirming. Cheers :wink:

**Comment by adrienlamotte** [comment]
@arturovt I have another problem, I think it's related to the fix you did.

Now the animation works fine on load, but when I navigate to another page (childreb route change) and then resize the page the mask disappear, with this message in the console : 

```GET http://myroute.local/subroute/#__lottie_element_24 net::ERR_ABORTED 404 (Not Found)```

Do you have any idea ? Maybe the setLocationHref need to be updated on route change ?

**Comment by adrienlamotte** [comment]
Well I tried to reproduce on a stackblitz, but it does seems I only have the issue inside on my project...
I have no clue, really strange issue...

I'll reopen the issue if I have more information ;)

**Comment by arturovt** [comment]
Hmm. I'm not sure how it should be updated. You can also do a little bit of reverse engineering, just open `node_modules/ngx-lottie/fesm2015/ngx-lottie.js` and start editing, it's safe. You can do anything there, add console logs etc. You can also find a place where `setLocationHref` is used and try to set there custom data. This will be helpful.

**Comment by orenagiv** [comment]
Hey @arturovt ,
It seems that this fix is not required anymore, and it's causing the animations to be rendered completely "black" on Safari.
Using lottie-web 5.6.8 and ngx-lottie 5.4.0.
Reproduced on Safari latest with Angular 9.

If I disable the "setPlayerLocationHref" function - everything works as expected again.
I reverted back to v4.1.0 and it works great.

**Comment by arturovt** [comment]
Thanks for information, I will check it up.

**Comment by arturovt** [comment]
Do you know what since what lottie-web version it’s not required anymore?

**Comment by orenagiv** [comment]
Hey @arturovt 
I think they made a change that fixes this issue on v5.4.0 of lottie-web.
See here:
https://github.com/airbnb/lottie-web/blob/master/CHANGELOG.md
Maybe this is related:
> FIX: added locationHref to gradient data
